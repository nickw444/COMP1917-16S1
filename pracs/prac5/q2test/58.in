58
-2 5 -9 6 -6 -8 -9 -3 8 3 8 4 -1 10 9 6 -8 -7 9 -6 -8 -7 7 2 6 3 -10 4 -8 -10 -9 1 8 -5 -7 4 -8 7 -7 10 7 -1 5 4 -2 5 -6 -7 -5 -3 7 -8 -6 -10 7 3 3 0
