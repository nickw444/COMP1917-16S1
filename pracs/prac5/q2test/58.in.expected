average of array: -0.431034
